### ISSUE TITLE
Odd behavior make_glpk_ilp.m

### ISSUE THREAD

**Issue by vicjang** [CTU-IIG/TORSCHE#2]
I noticed that there are two lines of code in [make_glpk_ilp.m](https://github.com/CTU-IIG/TORSCHE/blob/master/scheduling/contrib/make_glpk_ilp.m) that seems to have adverse effect.

```
93      copyfile([GLPKpath filesep 'contrib' filesep 'glpkmex' filesep 'doc' filesep 'glpkmex.m'], mexFileDestination);
94      copyfile([GLPKpath filesep 'contrib' filesep 'glpkmex' filesep 'doc' filesep 'glpkparams.m'], mexFileDestination);
```

If you look into those two files that are being copied (they are inside glpk-4.6.tar.gz), 
[glpkmex.m.txt](https://github.com/CTU-IIG/TORSCHE/files/119112/glpkmex.m.txt)
[glpkparams.m.txt](https://github.com/CTU-IIG/TORSCHE/files/119111/glpkparams.m.txt)
you will notice that the content are incomplete and they do not contain scripts that can be run in MATLAB.

Although the easiest way around all this is to use pre-built mex file, I would still suggest that these two lines be remove because they apparently do not serve their purposes here.

Thanks.

**Comment by dvoraj57** [comment]
Those two lines are there to be able to call 'help glpkparams' or 'help glpkmex'. That is why they are copied from the doc folder. Thus, those two lines have a reason to stay there. Although, they are not necessary.

**Comment by vicjang** [comment]
Thanks for replying. I understand what you meant, but this might cause a problem when the function `ilinprog` is called.

Here's a screenshot of the function call `glpkmex` in `ilinprog.m`

![image](https://cloud.githubusercontent.com/assets/1390467/13060035/560b915a-d3e3-11e5-9ce5-be73c2c62fd3.png)

If the aforementioned .m files are copied, the correct `glpkmex.m` will be overwritten, and `ilinprog` will not function.

At least that's what happened when I tested it. It got back working if I revert that file.

`glpkparams` didn't matter when I tested.
